downloads/downloads-api/src/main/java/com/duckduckgo/downloads/api/FileDownloader.kt

```diff
@@ -16,26 +16,25 @@
 
 package com.duckduckgo.downloads.api
 
-import androidx.annotation.WorkerThread
 import java.io.File
 import java.io.Serializable
 import android.os.Environment
+import androidx.annotation.AnyThread
 
 /** Interface for the starting point of a download. */
 interface FileDownloader {
 
     /** Starts a download. Takes as parameters a [PendingFileDownload] containing all details about the file to be downloaded and
      * a [DownloadCallback]. */
-    @WorkerThread
-    fun download(pending: PendingFileDownload, callback: DownloadCallback)
+    @AnyThread
```

Comment:
Update comment to:

Starts a download. Takes as parameters a [PendingFileDownload] containing all details about the file to be downloaded.

Reply:
will do